src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -152,7 +238,9 @@ const ChatItemMyTextV2 = ({
                 />
 
                 <View style={{flexDirection: 'row'}}>
-                  <Text style={messageStyle(true, messageType)}>{message}</Text>
+                  {attachments.length <= 0 && (
+                    <Text style={messageStyle(true, messageType)}>{message}</Text>
+                  )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on checking whether there are any attachments before rendering the message text. This prevents unnecessary rendering of empty text components.


<!-- This is an auto-generated comment by OSS CodeRabbit -->